packages/gatsby-cli/src/create-cli.ts

```diff
@@ -185,7 +185,12 @@ function buildLocalCommands(cli: yargs.Argv, isLocalSite: boolean): void {
     handler: handlerP(
       getCommandHandler(`develop`, (args: yargs.Arguments, cmd: Function) => {
         process.env.NODE_ENV = process.env.NODE_ENV || `development`
-        startGraphQLServer(siteInfo.directory, true)
+
+        // Only run recipes when admin is enabled
```

Comment:
I don't think this will work. The server also needs to run in the CLI when `gatsby recipes` is run.

Reply:
This is the develop command doesn't recipes use this one?
https://github.com/gatsbyjs/gatsby/blob/69a93baecb2837c36a67ad8fcdb626c29386ac80/packages/gatsby-cli/src/recipes.ts#L10-L20

Reply:
Ah, yes.